R/target_market_share.R

```diff
@@ -85,9 +85,9 @@ target_market_share <- function(data,
   if (by_company & weight_production) {
     warn(
       glue(
-        "`by_company` & `weight_production` shouldn't be both TRUE`:
-        You've supplied `by_company = TRUE` and `weight_production = TRUE`.
-        Did you forget to set one of these arguments to `FALSE`?"
+        "You've supplied `by_company = TRUE` and `weight_production = TRUE`.
+        This will result in company-level, weighted by the portfolio loan size.
+        Did you mean to set one of these arguments to `FALSE`?"
```

Comment:
@maurolepore let me know how you feel about this instead. Just want to give a bit of context as to "why" this might be cause for concern.

Reply:
I'm going to wait until you're up and have a look at this, but otherwise happy to merge this.

Reply:
I see little indication that this is a concern. As it's too strong to say that this _should not_ happen, how about saying that "this is rarely useful".

Reply:
That sounds good to me.